### ISSUE TITLE
Matching about PivotNet

### ISSUE THREAD

**Issue by toyot-li** [er-muyue/BeMapNet#8]
@er-muyue Hi, thanks for your great work. PDM elaborates on the point-level matching for PivotNet, but I did not find the details about how to do instance-level matching in the paper. 

Specifically, PDM assumes that "We consider the point matching problem between a predicted sequence Sˆ = {vˆn} and a ground truth sequence S^p = {v_n}", but what is your approach to assigning a ground truth sequence to a prediction?

Could you please explain more? Thanks.

**Comment by wenjie710** [comment]
Yes, PDM performs point-level matching between two instances. For instance-level matching, we employ a widely used bipartite matching approach, which quantifies dissimilarity between instances based on the lowest sequence matching cost.
